defender/tests.py

```diff
@@ -501,7 +501,6 @@ def test_admin(self):
         from .admin import AccessAttemptAdmin
         AccessAttemptAdmin
 
-    @patch('defender.config.PROTECTED_LOGINS', (ADMIN_LOGIN_URL, ))
     def test_decorator_middleware(self):
```

Comment:
@shin- there is still one of the middlewares. there was 2 originally, then I removed one. so there is still 1 left